File: Source/client-model/src/main/java/model/intention/ReadyForInitIntention.java
package model.intention;

import root.model.event.ClientIntention;

public class ReadyForInitIntention extends ClientIntention {

	public String roomName;

	public ReadyForInitIntention(String playerName, String roomName) {
		super(ClientIntentionType.ReadyForInit, playerName);

		this.roomName = roomName;
	}

}

File: Source/server/rabbit-game/src/SharedModel/ClientIntentions/ReadyForInitIntention.cs
namespace RabbitGameServer.SharedModel.ClientIntentions
{

	public class ReadyForInitIntention : ClientIntention
	{

		public String roomName { get; set; }

		public ReadyForInitIntention(string playerName, String roomName)
			: base(ClientIntentionType.ReadyForInit, playerName)
		{
			this.roomName = roomName;
		}
	}
}